stubs/inertia/resources/js/Pages/Auth/Login.vue

```diff
@@ -68,28 +68,20 @@
 
         data() {
             return {
-                remember: false,
                 form: this.$inertia.form({
                     email: '',
                     password: '',
-                    remember: ''
-                })
-            }
-        },
-
-        watch: {
-            remember(value) {
-                this.form.remember = value ? 'on' : ''
+                    remember: false
+                }).transform(data => ({
+                    ... data,
+                    remember: this.form.remember ? 'on' : ''
+                }))
```

Comment:
Personally I think it's better to put the `transform` in the `submit` handler. Just because it has more to do with sending the data to the server.